lib.php

```diff
@@ -18,6 +18,11 @@
 
 define('XAPI_REPORT_ID_ERROR', 0);
 define('XAPI_REPORT_ID_HISTORIC', 1);
+
+define('XAPI_TYPE_LIVE', 0);
+define('XAPI_TYPE_HISTORIC', 1);
+define('XAPI_TYPE_RECONCILE', 2);
+
```

Comment:
what about an error type?

Reply:
Errors go into the error table. This is more a type field to identify the import type.

Reply:
After looking at this again, it looks like there's a bit of crossover between these three (`XAPI_TYPE_`) and the ones above (`XAPI_REPORT_ID_`).
Let's discuss at our catchup

Reply:
@mlynn-lp can we change these to `XAPI_IMPORT_TYPE_` ?

Reply:
Yes I can do that.